test/functional/formatting_stack/linters/eastwood.clj

```diff
@@ -27,11 +27,11 @@
 
       "test-resources/eastwood_warning.clj"
       (matchers/in-any-order
-       [{:source              :eastwood/warn-on-reflection
```

Comment:
Maybe the linter rename is a breaking change 

OTOH any Eastwood upgrade can bring new warning types (e.g. `:eastwood/boxed-math`) so it to be expected that types can appear/disappear over time

Reply:
Well, it would if we had silencing / blacklisting implemented (see #132). So I think this'll be fine :)